corehq/apps/api/es.py

```diff
@@ -228,8 +228,8 @@ def post(self, request, *args, **kwargs):
         try:
             raw_post = request.body
             raw_query = json.loads(raw_post)
-        except Exception as ex:
```

Comment:
nit: why change to a single-letter variable name?

Reply:
```e``` is the most commonly used variable for an exception, so I'd like to stick to that convention.  Using other names has prevented me from finding these ```Exception.message``` usages earlier.